File: groceryManagerApp.java
import java.util.ArrayList;
import java.util.Scanner;

public class groceryManagerApp
{
   public static void main(String[] args)
   {
      Scanner input = new Scanner(System.in);
      Arraylist<String>ItemList = new Arraylist<>();
      
      
            System.out.print("\n Grocery List Manager");
            System.out.print("\n 1. Add item");
            System.out.print("\n 2. Remove item");
            System.out.print("\n 3. Show all available items");
            System.out.print("\n 0. Exit");
           
      
      
      while (true)
      {
            int choice = input.nextInt();
            String choice = input.nextLine;      
      switch(choice)
      {

                     case "1":
                              System.out.print("Enter item to add: ");
                              String addItem = input.nextLine();
                              ItemList.add(addItem);
                              SYstem.out.println(ItemList + " item has been added");
                              break;
                     
                     case "2":
                              System.out.print("Enter item to remove: ");
                              String removeItem = input.nextLine();
                              
                              if (ItemList.remove(removeItem));
                                 System.out.print("Item removed");

                              
                     case "3":
                             System.out.print("Show all available items");
                              
                     case "4":
                              System.out.print("Exiting>>>>>>>>>");
                              
                     default:
                              System.out.println("Choice unavailable");
            }
    }
  

}

File: grocery_manager_app.py
class grocery_manager_app :

   def   get_grocery(self):

         self.items = []
         
   def   add_item(self, item):

         self.items.append(item)
         
         print(f"'{item}' has been added.")
         
   def   remove_item(self, item):

         if item in self.items:
         
            self.items.remove(item)
            
            print(f"'{item}' removed from list")
            
         else:
         
               print(f"'{item}' not available")
               
               
   def   show_item(self):

         if not_self.items:
         
            print("Your list is empty")
            
         else: 
         
               print("Grocery List Items")
               
               for count, item in (self.items):
               
                     print(f"{count}.{item}")
                     
                        
   def main():
      
            
         
         while True:
               
               print("\n Grocery List Manager")
               print("\n 1. Add item")
               print("\n 2. Remove")
               print("\n 3. Show items")
               print("\n 0. Exit")
               
               choice = input("Enter your choice: ")
            
               if choice == '1':
               
                  item = input("Enter item to add: ")
                  manager.add_item(item)
                  
               elif choice == '2':
               
                  item = input("Enter item to remove: ")
                  manager.show_items()
                  
               elif choice == '3':
               
                    print("Here are the items available")
                  
               elif choice == '4':
               
                  print("Exiting >>>>>>>>>>>")
                  print("Goodbye!!")
                  
                  
               else:
               
                  print("Choice is not oavailable. Try again.")
            
   main()
         
